src/table/TableBodyWrapper.jsx

```diff
@@ -57,18 +90,43 @@ const TableBodyWrapper = ({
     colIdx: -1,
     isEnabled: selectionsEnabled,
   });
+  const [isScrollbar, setIsScrollbar] = useState(false);
 
   useEffect(() => {
     selDispatch({ type: 'set-enabled', payload: { isEnabled: selectionsEnabled } });
     setColumnRenderers(getColumnRenderers);
   }, [selectionsEnabled, columns.length]);
 
+  // Setup isScrolling variable
+  let isScrolling;
+  // eslint-disable-next-line consistent-return
   useEffect(() => {
     addSelectionListeners(selectionsAPI, selDispatch, setShouldRefocus);
+    // to display scrolling bar when the user scrolls in windows
+    // if (navigator.appVersion.indexOf('Win') !== -1) {
+    const handleScroll = () => {
+      isScrollbar === false && setIsScrollbar(true);
+      // Clear our timeout throughout the scroll
+      window.clearTimeout(isScrolling);
+      // Set a timeout to run after scrolling ends
+      isScrolling = setTimeout(() => setIsScrollbar(false), 100);
+    };
+    window.addEventListener('scroll', handleScroll, true);
+    return () => window.removeEventListener('resize', handleScroll);
+    // }
   }, []);
 
   return (
-    <TableBody className={`${classes.body} ${classes.cellBase}`}>
+    <TableBody
+      className={`${classes.body} ${classes.cellBase} ${
+        // in windows, make all scrollbar button, track and thumb transparent
+        // navigator.appVersion.indexOf('Win') !== -1 &&
+        classes.hideScrollbar
+      }  ${
+        // navigator.appVersion.indexOf('Win') !== -1 &&
+        isScrollbar && classes.showScrollbar
```

Comment:
Comments here are just cluttering, I think the classes are self explanatory.

Reply:
Would this work?
```suggestion
        ${isScrollbar ? classes.showScrollbar : classes.hideScrollbar}
```

Reply:
good. updated